src/blog/_posts/2018-05-09-opera-53.md

```diff
@@ -0,0 +1,234 @@
+
+        // Start a fetch controlled by the above controller.
+        fetch(url, { signal }).then(...);
+
+        // Abort the fetch.
+        controller.abort();
+
+### Blink > File API
+
+* Attempting to read from an invalid or non-existing Blob URL now results in a
+  network error instead of a 404 error.
+
+### Blink > Forms
+
+* The `<textarea>` and `<select>` elements now support the `autocomplete`
+  attribute.
+* Toggling a mutable checkbox now triggers a sequence of three events: `click`,
+  `input` and `change`, in that order. Previously, the `input` event was not
+  fired.
+
+### Blink > Fullscreen
+
+* If a page in fullscreen mode uses `window.focus()` to switch focus to another
+  page, fullscreen mode is exited.
+
+### Blink > GetUserMedia
+
+* The new [`MediaStreamTrack.getCapabilities()`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/getCapabilities)
+  returns a `MediaTrackCapabilities` object with details about the browser's
+  capabilities.
+
+### Blink > JavaScript
+
+* The [`Function.prototype.toString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/toString)
+  function now returns the function's source code exactly as it was written,
+  including white-space and comments.
+* JSON is now a [syntactic subset of ECMAScript](https://github.com/tc39/proposal-json-superset),
+  which allows line separator (U+2028) and paragraph separator (U+2029)
+  characters in string literals.
+* A `try`/`catch` statement can now [omit the binding parameter](https://tc39.github.io/proposal-optional-catch-binding/).
+  In other words, if the actual exception is not interesting, the short form
+  `try { ... } catch { ... }` can be used.
+* `String.prototype.trimStart()` and `String.prototype.trimEnd()` are now
+  supported for trimming leading and trailing white-space characters from
+  strings, complementing the existing `String.prototype.trim()`. The
+  non-standard `trimLeft()`/`trimRight()` functions remain as aliases for
+  backwards compatibility.
+* `Array.prototype.values()` is now supported. It returns an iterator over the
+  values at each index in the array.
+
+### Blink > Layout
+
+*   The `grid-` prefix has been removed from the CSS gutter properties `grid-gap`,
+    `grid-row-gap` and `grid-column-gap`, whose names now become just `gap`,
+    `row-gap` and `column-gap`.
+
+    The default value is `normal` for all three. The old names remain as aliases
+    for the new names. The property `column-gap` already existed as part of CSS
+    Multi-column Layout.
+*   Elements with a transform property and with the `display` property set to
+    `table-row`, `table-row-group`, `table-header-group`, `table-footer-group`,
+    `table-cell` or `table-caption`, are now containing blocks for fixed
+    position elements.
+
+### Blink > Media
+
+* `autoplay` is now [only allowed](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes)
```

Comment:
This section could be clearer.

Reply:
Actually, I was going to comment on this (but of course forgot.) I don't know if this applies to Opera at all. For instance, in Chrome, there's chrome://media-engagement, but Opera doesn't have that. So I mean to investigate if these policies changes even affect us, and if not, just drop the section.

Assuming we will include it; how would you change it to make it more clear?

Reply:
I've investigated and tested, and it seems O53's behavior has changed according to this. Or at least I've confirmed that muted video will autoplay when non-muted will not, and that it makes a difference if you navigate directly to a page with autoplaying video or if you follow an "intra-site" link to the page.

Testing the media engagement index stuff turned out to be difficult. Could not really make that "work" in Chrome either.

Reply:
Oh, and I also rephrased this point. Let me know if it became more or less clear as a result. :-)

Reply:
Better!